deployments/liqo_chart/crds/sharing.liqo.io_advertisements.yaml

```diff
@@ -37,8 +39,13 @@ spec:
           description: AdvertisementSpec defines the desired state of Advertisement
```

Comment:
Personally, I would avoid the name of the object in the description.
E.g., 
AdvertisementSpec defines the desired state of Advertisement
==> Defines the desired state of Advertisement

Reply:
I didn't know. Got it.
Fine for me. Thanks!